creation/lib/cvWParamDict.py

```diff
@@ -1273,8 +1277,8 @@ def populate_gridmap(params, gridmap_dict):
 #####################################################
 # Populate security values
 def populate_main_security(client_security, params):
-    if params.security.proxy_DN is None:
-        raise RuntimeError("DN not defined for classad_proxy")
+    # if params.security.proxy_DN is None:
+    #    raise RuntimeError("DN not defined for classad_proxy")
     client_security["proxy_DN"] = params.security.proxy_DN
```

Comment:
Probably the assignment should be conditional to the proxy being there.
There may be some compensation downstream needed if `client_security["proxy_DN"]` is empty. 
Check if it should do None or "".

Reply:
implemented